config/core/deployments/controller.yaml

```diff
@@ -19,6 +19,7 @@ metadata:
   namespace: knative-serving
   labels:
     serving.knative.dev/release: devel
+    knative.dev/high-availability: "true"
```

Comment:
Can we put this on the Pods as well?

Reply:
Yeah, sure, but why? Do you have other plans for this label than only using it in the operator?